lte/gateway/c/li_agent/main.cpp

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <lte/protos/mconfig/mconfigs.pb.h>
+#include <thread>
+
+#include "MagmaService.h"
+#include "MConfigLoader.h"
+#include "ServiceRegistrySingleton.h"
+
+#include "InterfaceMonitor.h"
+#include "PDUGenerator.h"
+#include "ProxyConnector.h"
+
+#define LIAGENTD "liagentd"
+#define LIAGENTD_VERSION "1.0"
+
+static magma::mconfig::LIAgentD get_default_mconfig() {
+  magma::mconfig::LIAgentD mconfig;
+  mconfig.set_log_level(magma::orc8r::LogLevel::INFO);
+  return mconfig;
+}
+
+static magma::mconfig::LIAgentD load_mconfig() {
+  magma::mconfig::LIAgentD mconfig;
+  magma::MConfigLoader loader;
+  if (!loader.load_service_mconfig(LIAGENTD, &mconfig)) {
+    MLOG(MERROR) << "Unable to load mconfig for liagentd, using default";
+    return get_default_mconfig();
```

Comment:
Is this default fairly meaningful (likely to be useful)? Or should we die here.

Reply:
Yeah default is fine, its just used for logging level, no vital info is in mconfig